CHANGELOG.md

```diff
@@ -12,6 +12,39 @@ Bugfixes:
 
 Other improvements:
 
+## [v5.0.0](https://github.com/purescript/purescript-prelude/releases/tag/v4.1.0) - 2021-MONTH-DATE
+
+Breaking changes:
+- Make library compile on `v0.14.0` (#206)
+- `purescript-proxy` was ported to this repo (#230)
+- `purescript-generics-rep` was ported to this repo (#235)
+- Move the `Applicative` Superclass law from `Monad` to `Bind` and rename it
+to the `Apply` Superclass law (#229)
+
+New features:
+- Added `Bounded` instance for records (#208)
+- Added `Show` instances to `Data.Generics.Rep` types (#250)
+
+Bugfixes:
+- No longer use reference equality check in `Array`'s `Eq` instance because it breaks referential transparency (#187).
+- Fix Ring laws (#228)
+
+Other improvements:
+- Added `lift2` example using `Maybe` (#213)
+- Added `const` example (#214)
+- Added `power` example (#253)
+- Improve documentation in various things (#217)
+    - Clarify `Array`'s `do notation`
+    - Clarify purpose of `Monoid` and `Semigroup` newtypes
+- Clarify `Unit` representation in FFI code (#223)
```

Comment:
I don't know if this level of detail (where each PR is mentioned) is necessary for changelogs. It would probably be fine to just write something like "Added more examples" or not even mention that, since it doesn't really affect existing users of the library.
I think the main purpose of the changelog is to help folks quickly review API changes since their last update.

Reply:
I'm good with making this change if others agree. I thought "Added more docs" or something wasn't that helpful either.

Reply:
I think it's reasonable to point out documentation updates, but I agree they ought to be as concise as possible. For example:

```
Other improvements:
- Added examples to documentation comments for `lift2`, `const`, and `power` (#213, #214, #253)
- Improved documentation for `do` notation for `Array`, the purpose of the `Monoid` and `Semigroup` newtypes, and the representation of `Unit` in FFI code (#217, #223)
```

Reply:
(They're already de-prioritized by being bundled under the "Other improvements" section, which is easier to skim if you're only looking for API changes)

Reply:
I disagree, I think almost every PR should be referenced in the changelog. It's not as if there's a massive amount of detail to wade through here, and it's important that we provide credit to contributors for the changes of theirs we've merged.

Reply:
Personally I think having separate bullet points for each of the separate documentation PRs (as you do currently) is ideal. We can group them all under a "Documentation improvements" heading so that people who aren't interested in anything that doesn't affect the actual API can easily skim past them.

Reply:
I agree almost all pull requests should be referenced in the changelog. I'm also fine with bullet points for each documentation PR.

As a note, the more concise version I added doesn't drop any PRs, it just allows some of them to be included in the same line if they're logically related. I also think _that's_ fine, but we should probably have an agreed-upon default.